src/runtime/pkg/containerd-shim-v2/create.go

```diff
@@ -243,6 +243,11 @@ func loadRuntimeConfig(s *service, r *taskAPI.CreateTaskRequest, anno map[string
 }
 
 func checkAndMount(s *service, r *taskAPI.CreateTaskRequest) (bool, error) {
+	// In the confidential computing scenario, there is no Image information on the host,
+	// so there is no Rootfs.
+	if len(r.Rootfs) == 0 {
```

Comment:
Should there be a check to verify if `image.ServiceOffload` is set to true ?

Reply:
I got it, it commited by https://github.com/kata-containers/kata-containers/commit/539afba03daa341474a6223a852bed09eb835bc4. 
I will fix.

Reply:
I have a question, does a host machine need to support starting different instances? 
In other words, do CC instances and Kata instances need to be mixed on the same machine?

Reply:
> I have a question, does a host machine need to support starting different instances? In other words, do CC instances and Kata instances need to be mixed on the same machine?

Not for now.

Reply:
fixed